index.js

```diff
@@ -25,28 +27,33 @@ module.exports = function (fileName, converter) {
       return; // ignore
     }
     if (file.isStream()) {
+	  skipConversion = true;
       return this.emit('error', new PluginError('gulp-jsoncombine', 'Streaming not supported'));
     }
     try {
       data[file.relative.substr(0,file.relative.length-5)] = JSON.parse(file.contents.toString());
```

Comment:
I didn't want to break backwards compatibility, but I noticed this schema to create keys can be a bit confusing and can lead to conflicts.

```
hello.txt => hell
hellx.txt => hell
```